### PR TITLE
fix PipelineLink url

### PR DESCRIPTION
**Pull request by pipipipipi43**
#### What type of this PR

/kind bug

#### What this PR does / why we need it:

When there is a task that is already being executed in the pipeline, the link error of clicking the execution prompt is wrong
image
Steps to reproduce:
Corresponding to the existing branch of yml, click to create a new pipeline when executing, an error message is displayed, and the copy link cannot be opened

### CODE REVIEW

File: modules/pipeline/commonutil/linkutil/link.go [hunk 1]
```diff
@@ -15,6 +15,7 @@ package linkutil
 
 import (
 	"fmt"
+	"github.com/erda-project/erda/modules/pipeline/conf"
```
Comment (Effet) [inline modules/pipeline/commonutil/linkutil/link.go:18]: import order

**Review by Effet — CHANGES_REQUESTED** [review]

File: modules/pipeline/commonutil/linkutil/link.go [hunk 2]
```diff
@@ -45,15 +46,7 @@ func GetPipelineLink(bdl *bundle.Bundle, p spec.Pipeline) (bool, string) {
...
-	// get domain protocol
-	clusterInfo, err := bdl.QueryClusterInfo(p.ClusterName)
-	if err != nil {
-		return false, ""
-	}
-	protocol := clusterInfo.Get(apistructs.DICE_PROTOCOL)
-	if protocol == "" {
-		return false, ""
-	}
+	Domain := conf.UIDomain()
```
Comment (kakj-go) [inline modules/pipeline/commonutil/linkutil/link.go:49]: Domain -> domain

File: modules/pipeline/commonutil/linkutil/link.go [hunk 3]
```diff
@@ -45,15 +46,7 @@ func GetPipelineLink(bdl *bundle.Bundle, p spec.Pipeline) (bool, string) {
...
 
-	return true, fmt.Sprintf("%s://%s/workBench/projects/%d/apps/%d/pipeline/%d", protocol, org.Domain, projectID, appID, p.ID)
+	return true, fmt.Sprintf("https://%s/%s/dop/projects/%d/apps/%d/pipeline?pipelineID=%d", Domain, org.Name, projectID, appID, p.ID)
```
Comment (Effet) [inline modules/pipeline/commonutil/linkutil/link.go:51]: should move link fmt to dop.

**Review by Effet — CHANGES_REQUESTED** [review]

File: modules/pipeline/commonutil/linkutil/link.go (hunk 3 above)
Comment (Effet) [reply modules/pipeline/commonutil/linkutil/link.go]: and hard-code `https` is very bad

**Pull request closed without merging**